py/rackattack/tcp/allocation.py

```diff
@@ -136,7 +136,7 @@ def _allocationEventBroadcasted(self, event):
         if eventType == "changedState":
             self._waitEvent.set()
         elif eventType == "providerMessage":
-            logging.info("Rackattack provider says: %(message)s", dict(message=event['message']))
+            logging.debug("Rackattack provider says: %(message)s", dict(message=event['message']))
```

Comment:
This should not be debug. It has useful information. Perhaps consult with Michael.